### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by HugoFerrant**
see https://github.com/api-platform/api-platform/issues/1816

### CODE REVIEW

File: client-generator/troubleshooting.md [hunk 1]
```diff
@@ -55,3 +55,7 @@ cause: null } }
 Check access to the specified URL, in this case `https://demo.api-platform.com/contexts/Entrypoint`, use curl to check
 access and the response `curl https://demo.api-platform.com/contexts/Entrypoint`. In the above case an "Access Denied"
 message in JSON format was being returned.  
+
+## Docker distribution on windows and hot-reloading
```
Comment (alanpoulain) [inline client-generator/troubleshooting.md:59]:
```suggestion
## Docker distribution on Windows and hot-reloading
```

File: client-generator/troubleshooting.md [hunk 2]
```diff
@@ -55,3 +55,7 @@ cause: null } }
...
+
+Due to [a long-time know Docker for Windows issue](https://forums.docker.com/t/file-system-watch-does-not-work-with-mounted-volumes/12038), the files changes on the host are not notified on the pwa container. This cause the hot-reloading feature to not working for windows users.
```
Comment (alanpoulain) [inline client-generator/troubleshooting.md:61]:
```suggestion
Due to [a long-time known Docker for Windows issue](https://forums.docker.com/t/file-system-watch-does-not-work-with-mounted-volumes/12038), the files changes on the host are not notified on the `pwa` container. It causes the hot-reloading feature to not working properly for Windows users.
```

File: client-generator/troubleshooting.md (hunk 1 above)
Comment (HugoFerrant) [reply client-generator/troubleshooting.md:59]: Yup thx, there is another one at the end too.

File: client-generator/troubleshooting.md (hunk 2 above)
Comment (HugoFerrant) [reply client-generator/troubleshooting.md:61]: perfect ! Thx !

**Review by alanpoulain — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks @HugoFerrant

**Pull request merged**